magda-migrator-content-db/sql/V1_8__footer_default_config.sql

```diff
@@ -0,0 +1,43 @@
+INSERT INTO "public"."content" ("id", "type", "content") VALUES ('footer/navigation/categories/medium/data.gov.au', 'application/json', '{
+  "label": "Data.gov.au",
+  "links": [
+    [
+      "About",
+      "page/about"
+    ],
+    [
+      "Suggest a dataset",
```

Comment:
Remove these links for internal magda? See #1782

Reply:
I'd probably also separate links and categories into two separate items. Would make migrations like #1782 easier. What do you think @AlexGilleran ?

Reply:
- I thought `Suggest a dataset` will stay for our generic version but removed for internal Magda?
If not the case, I can simply remove it 😄 
- Thought about break the items into two levels: `categories` then `links` but the problem will be hard to maintain two `order` information. Right now, `order` field is for the order of `categories` and links order is determined by array item order. Also, try to make data structure similar to the current one so less code to edit. @nahidakbar Let me know if you got a better idea on the data structure 😄

Reply:
Yeah I would've thought having them together is a bit easier, but I don't have a strong opinion 🤷‍♂️